src/index.html

```diff
@@ -11,7 +11,146 @@
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth:
+          <input
+          type="date"
+          name="datebith"
+          >
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-set">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          minlength="5"
+          maxlength="20"
+          required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+          type="password"
+          name="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-set">
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          <label>
+            Do you love cats?
+            <input
+            type="radio"
+            name="catlike"
+            value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="catlike"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+        <label class="form-field">
+            What is your favorite color?
+          <input
+          type="color"
+          name="favorite-color"
+          value="#000"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="timesleep">
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+          name="favorite-car"
+          multiple
+          required
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+            <option value=" "></option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input
+          type="range"
+          name="work-range"
+          value="0"
+          min="0"
+          max="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-set">
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="comments"></textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recomendation">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+    </form>
+    <button type="submit">Submit</button>
```

Comment:
The submit button should be inside the `<form>` tag, please move it inside the form so that it properly submits the form data.